src/Entity/OgRole.php

```diff
@@ -254,7 +254,8 @@ public function set($property_name, $value) {
       'group_type',
       'group_bundle',
     ]);
-    if ($is_locked_property && !$this->isNew()) {
+    $value_has_changed = !empty($original_value = $this->get($property_name)) && $original_value != $value;
+    if ($is_locked_property && !$this->isNew() && $value_has_changed) {
       throw new OgRoleException("The $property_name cannot be changed.");
```

Comment:
This exception was being thrown during tests, turned it was thrown when an existing role was resaved without changing anything :)